### PR TITLE
Set up resque and capistrano for stage worker machine

### PR DESCRIPTION
**Pull request by tallenaz**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
+require 'resque/server'
+
 Rails.application.routes.draw do
   resources :catalog, param: :druid, only: %i[create update]
   resources :moab_storage, only: %i[index show]
 
+  mount Resque::Server.new, at: "/resque"
```
Comment (ndushay) [inline config/routes.rb]: do you want to make this conditional?  like behind a feature flag?  Or does it not matter if we add this all the time.

File: config/deploy/stage.rb [hunk 2]
```diff
@@ -1,7 +1,10 @@
 server 'preservation-catalog-stage.stanford.edu', user: 'pres', roles: %w[app db web]
+server 'preservation-worker-stage.stanford.edu', user: 'pres', roles: %w[app db web]
 
 Capistrano::OneTimeKey.generate_one_time_key!
 set :rails_env, 'production'
 set :bundle_without, 'deploy test'
 set :deploy_to, '/opt/app/pres/preservation_catalog'
-append :linked_files, "config/newrelic.yml"
+
+# uncomment if/when we turn newrelic on
```
Comment (ndushay) [inline config/deploy/stage.rb]: 👍

File: config/routes.rb (hunk 1 above)
Comment (tallenaz) [reply config/routes.rb]: I found one way to do this is with Rails route constraints, like:

`mount Resque::Server.new, at: '/resque', constraints: { host: /worker-machine-name-here/ }`

**Comment by tallenaz** [comment]
Closing this in favor of an upcoming fresh PR. From there, I'll link back to this.

**Pull request closed without merging**